src/components/collapsible/Collapsible.scss

```diff
@@ -38,14 +38,40 @@
         width: 100%;
         padding: 10px 0;
         text-align: left;
+
+        + .bdl-collapsible-actionItems {
+            display: none;
+        }
+
+        &:focus,
+        &:hover {
+            text-decoration: underline;
+
+            + .bdl-collapsible-actionItems {
+                display: block;
+            }
+        }
     }
 
     .collapsible-card-content {
         padding: 10px 0;
     }
 
-    &.is-open .collapsible-card-header-caret {
-        transform: rotateZ(180deg);
+    &.is-open {
+        .collapsible-card-header-caret {
+            transform: rotateZ(180deg);
+        }
+
+        .collapsible-card-title {
+            &:focus,
+            &:hover {
+                text-decoration: none;
+            }
```

Comment:
btw you can make a css mixin with this, and use it both places instead of duplicating

Reply:
The almost similar css block above has  `+ .bdl-collapsible-actionItems { display: block }` inside the focus and hover. So they are not entirely  same to make mixin.